startup_runners/startup_runner.c

```diff
@@ -13,19 +13,20 @@
 #include <errno.h>
 #include <dlfcn.h>
 #include <err.h>
+#include <stdint.h>
 #include <unistd.h>
 
-#define BENCH_FUNC_NAME "run_iter"
+#include "libkruntime.h"
 
-/* from libkruntime */
-double clock_gettime_monotonic();
+#define BENCH_FUNC_NAME "run_iter"
 
 int
 main(void)
 {
     double    start_time = -1;
 
-    start_time = clock_gettime_monotonic();
+    krun_measure(0);
```

Comment:
Should we use mdata_index of 1 to ensure wallclock timing is innermost?

This will screw up unfortunately, as the zero'th mdata_index will contain junk and upset the integrity checks inside krun_measure.

Sigh. Sorry, this is my fault.